pytorch_lightning/plugins/sharded_plugin.py

```diff
@@ -36,15 +36,14 @@ def configure_ddp(
             self, model: LightningModule, device_ids: List[int]
     ):
         self._wrap_optimizers(model)
-        return LightningShardedDataParallel(model, sharded_optimizer=model.trainer.optimizers)
+        model = LightningShardedDataParallel(model, sharded_optimizer=model.trainer.optimizers)
+        model.device_ids = device_ids
```

Comment:
I think this is fine, how about `self.device_ids` and keeping track within the plugin rather than the model? Same with ddp_plugin.py

Reply:
Done !